backend/src/create.rs

```diff
@@ -360,7 +361,7 @@ fn apply_existing_filters(
             let pt = map
                 .get_r(r)
                 .linestring
-                .line_interpolate_point(percent)
+                .point_at_ratio_from_start(&Euclidean, percent)
```

Comment:
This is the new api from https://github.com/georust/geo/pull/1321 - it's more explicit about metric space, and also has some new APIs which you can see further down.